### PR TITLE
Fixed query to find pending executions

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR fixes the query for `findPending` and make it independent from timezone (by using `now` and give the decision to the database).

### CODE REVIEW

File: src/Entity/TaskExecutionRepository.php [hunk 1]
```diff
@@ -129,17 +129,13 @@ public function findByTaskUuid($taskUuid)
     /**
      * {@inheritdoc}
      */
-    public function findScheduled(\DateTime $dateTime = null)
```
Comment (wachterjohannes) [inline src/Entity/TaskExecutionRepository.php]: the interface has changed a few time ago. so i have removed it also from this class.

File: tests/Functional/Entity/TaskExecutionRepositoryTest.php [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->taskExecutionRepository = self::$kernel->getContainer()->get('task.storage.task_execution');
+        $this->taskRepository = self::$kernel->getContainer()->get('task.storage.task');
+    }
+
+    public function testSave(TaskInterface $task = null, \DateTime $scheduleTime = null, $status = TaskStatus::PLANNED)
```
Comment (danrot) [inline tests/Functional/Entity/TaskExecutionRepositoryTest.php]: Do you really think it is a good idea to reuse the test method? Having the assertions also executed in the other tests feels like something that might bite us one day.

Comment (wachterjohannes) [inline tests/Functional/Entity/TaskExecutionRepositoryTest.php]: but this is a "prerequisite" for the other testcases but i could also extract it to another function.

Comment (danrot) [inline tests/Functional/Entity/TaskExecutionRepositoryTest.php]: I would exclude it in another function... There is no need that these tests are executed multiple times within a single run.

**Comment by wachterjohannes** [comment]
@danrot fixed.

**Pull request merged**